lib/ansible/modules/source_control/git.py

```diff
@@ -611,23 +604,39 @@ def get_head_branch(git_path, module, dest, remote, bare=False):
     if os.path.isfile(repo_path):
         try:
             git_conf = open(repo_path, 'rb')
```

Comment:
This is a forgotten, unclosed and unnecessary file descriptor. We need to wipe this line.